lib/compile/bundles/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+import most from "most";
+
+import resolveAsset from "../modules/resolve";
+import loadAst from "../modules/load-ast";
+
+export default function (compilation) {
+  return most
+    .from(compilation.opts.emit)
+    .filter(def => def.entry || def.split)
+    .map(function bootstrapBundle (bundleDef) {
```

Comment:
Yea I figured that's why. I actually assumed that you ran into a bug and this was a bit of instrumentation you had done. As far as having multiple frames under the same name, no it doesn't matter but you dont want to have this either.

filename1.js bootstrapBundle ln10
filename2.js bootstrapBundle ln100
filename3.js bootstrapBundle ln13

As far as stack traces go, I start with the file name and line number 90% of the time but I have no idea what a stack from most looks like. You're also using promises in a few spots so between the two you may need all the help you can get. One approach would be to standardize on fat arrow and add a name every time you debug an area and find it helpful.